src/mds/MDSDaemon.cc

```diff
@@ -283,7 +283,7 @@ void MDSDaemon::set_up_admin_socket()
   ceph_assert(r == 0);
   r = admin_socket->register_command("scrub start "
 				     "name=path,type=CephString "
-				     "name=scrubops,type=CephChoices,strings=force|recursive|repair,n=N,req=false "
+				     "name=scrubops,type=CephChoices,strings=force|recursive|repair|eval_stray_at_root,n=N,req=false "
```

Comment:
Let's call this new flag `scrub_mdsdir` and document that providing this flag triggers stray reintegration.

Reply:
done